docs/static/modules.asciidoc

```diff
@@ -0,0 +1,118 @@
+[[logstash-modules]]
+== Working with Logstash Modules
+
+Logstash modules provide a quick, end-to-end solution for ingesting data and
+visualizing it with purpose-built dashboards.
+
+Each module comes pre-packaged with Logstash configurations, Kibana dashboards,
+and other meta files that make it easier for you to set up the Elastic Stack for
+specific use cases or data sources.
+
+[float]
+[[running-logstash-modules]]
+=== Running modules
+
+When you run a module, Logstash creates and loads the pipeline configurations
+required to read and parse the data. It also loads the index pattern,
+field definitions, searches, visualizations, and dashboards required to
+visualize your data in Kibana.
+
+To run a module, you use the `--modules` option:
+
+[source,shell]
+----
+bin/logstash --modules MODULE_NAME [-M CONFIG_SETTINGS]
+----
+
+
+//TODO: For 6.0, show how to run mutliple modules
+
+Where `MODULE_NAME` is the name of Logstash module and `CONFIG_SETTINGS`
+is one or more optional configuration settings. `CONFIG_SETTINGS` are only
+required when the default configuration doesn't meet your needs, or you need to
+override settings specified in the `logstash.yml` settings file.
+
+For example, the following command runs the Netflow module with the default
+settings:
+
+[source,shell]
+----
+bin/logstash --modules netflow
+----
+
+The following command runs the Netflow module and overrides the Elasticsearch
+`host` setting:
+
+[source,shell]
+----
+bin/logstash --modules netflow -M "netflow.var.elasticsearch.host=es.mycloud.com"
```

Comment:
@suyograo missed this one initially, but this should be plural "hosts" right? and its also missing the port?